src/components/spatial/tiling-utils.js

```diff
@@ -1,15 +1,16 @@
+
+export function tileToBoundingBox(x, y, z, maxHeight, maxWidth, tileSize) {
+  return {
```

Comment:
```suggestion
  return {
    const zFactor = 2 ** (-1 * z);
```
or something like that, and then the line below is:
```
west: x * tileSize * zFactor
```